.github/workflows/publish-docs.yml

```diff
@@ -20,11 +20,13 @@ jobs:
         uses: actions/setup-node@04c56d2f954f1e4c69436aa54cfef261a018f458 # renovate: tag=v2.5.0
         with:
           node-version: ${{ env.node-version }}
+          cache: 'yarn'
 
       - name: Set up Python ${{ env.python-version }}
         uses: actions/setup-python@f38219332975fe8f9c04cca981d674bf22aea1d3 # renovate: tag=v2.3.1
         with:
           python-version: ${{ env.python-version }}
+          cache: 'pip'
 
       - name: Init platform
         id: init
```

Comment:
We still have some mentions of the word `cache` in the code below:

```yaml
        run: |
          echo "::set-output name=yarn_cache::$(yarn cache dir)"
          python -c "from pip._internal.locations import USER_CACHE_DIR; print('::set-output name=pip_cache::' + USER_CACHE_DIR)"
          git config --global core.autocrlf false
          git config --global core.symlinks true
          git config --global user.email 'bot@renovateapp.com'
          git config --global user.name  'Renovate Bot'
          python --version
          pip --version
```

Do I need to change anything to this code?

Reply:
I think the forst two lines can be removed from that script

Reply:
I've removed the first two lines from the script. 😉